dbexport.php

```diff
@@ -67,7 +67,11 @@
         $data = $DB->get_records($table, null, 'id ASC');
 
         // Use Moodle's dataformatting functions to output the data in the desired format.
-        download_as_dataformat($exportfile, $dataformat, array_keys($DB->get_columns($table)), $data);
+        if ($CFG->branch >= 39){
```

Comment:
This works, but can we please refactor the code to use the handle_deprecations class found under /utilities.